sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_client.py

```diff
@@ -272,12 +272,16 @@ def get_deleted_certificate(self, certificate_name, **kwargs):
     @distributed_trace
     def purge_deleted_certificate(self, certificate_name, **kwargs):
         # type: (str, **Any) -> None
-        """Permanently deletes the specified deleted certificate.
+        """Permanently delete a certificate. Possible only in vaults with soft-delete enabled.
 
         Performs an irreversible deletion of the specified certificate, without
         possibility for recovery. The operation is not available if the recovery
-        level does not specified 'Purgeable'. This operation requires the
-        certificate/purge permission.
+        level does not specified 'Purgeable'. This method is only necessary for
+        immediate purging of a certificate, as a certificate will still eventually
+        purge on its :py:attr:`~azure.keyvault.certificates.DeletedCertificate.scheduled_purge_date`
+        without calling this method.
```

Comment:
- suggest "... necessary for purging a certificate before its :py:attr:.."
- we could link to the recovery level property (also clarifies what's meant by "__the__ recovery level")
- does not ~specified~ specify

Reply:
sounds a lot better, thanks!